views/home.erb

```diff
@@ -117,6 +118,19 @@
                     <%= event.start_time.in_time_zone.strftime('%m/%d/%Y @ %l:%M %P') %><br>
                     <%= event.location.name %>
                 </td>
+                <% if event.trainer_confirmed == 1 %>
+                    <td class="table-actions">
+                        <div class="dcf-input-checkbox">
+                            <input disabled id="training-confirmed" title="Training Confirmed" type="checkbox" <%= 'checked' %> name="training_confirmed">
```

Comment:
This whole `<%= 'checked' %>` part can be replaced with `checked` similar to how you can disable it by just adding `disabled`.